exporters/otlp/include/opentelemetry/exporters/otlp/otlp_grpc_exporter.h

```diff
@@ -11,26 +11,56 @@
 
 #include "opentelemetry/sdk/trace/exporter.h"
 
+#include "opentelemetry/sdk/common/env_variables.h"
+
 OPENTELEMETRY_BEGIN_NAMESPACE
 namespace exporter
 {
 namespace otlp
 {
+
+inline const std::string GetOtlpGrpcDefaultEndpoint()
+{
+  constexpr char kOtlpGrpcEndpointEnv[]     = "OTEL_EXPORTER_OTLP_GRPC_ENDPOINT";
+  constexpr char kOtlpGrpcEndpointDefault[] = "localhost:4317";
+
+  auto endpoint = opentelemetry::sdk::common::GetEnvironmentVariable(kOtlpGrpcEndpointEnv);
+  return endpoint.size() ? endpoint : kOtlpGrpcEndpointDefault;
+}
+
+inline const bool GetOtlpGrpcDefaultIsSslEnable()
+{
+  constexpr char kOtlpGrpcIsSslEnableEnv[] = "OTEL_EXPORTER_OTLP_GRPC_SSL_ENABLE";
+  auto ssl_enable = opentelemetry::sdk::common::GetEnvironmentVariable(kOtlpGrpcIsSslEnableEnv);
+  if (ssl_enable == "True" || ssl_enable == "TRUE" || ssl_enable == "true" || ssl_enable == "1")
```

Comment:
Convert it to lower case before comparison, to make the comparison case insensitive?

Reply:
Good point will change that.

Reply:
Ok. I realize there is no method available in `stdlib` to perform this conversion. we have to iterate over the string to convert it.  The existing comparison looks simple as compared to that :)